docs/intro_testers.rst

```diff
@@ -0,0 +1,61 @@
+.. _intro_testers:
+
+========================
+Introduction for Testers
+========================
+
+This document is aimed at providing an introduction to running existing test suites.
+
+We assume here that you have access to an operational test lab; if not, ask
+your local admin for access!
+
+If you're here to test upstream Ceph, start `here
+<http://ceph.github.io/sepia/adding_users/>`__.
+
+
+Terminology
+===========
+
+In the abstract, each set of tests is defined by a `suite`. All of our suites
+live in the `ceph-qa-suite repository
+<https://github.com/ceph/ceph-qa-suite/>`__, in the `suites` subdirectory.
+
```

Comment:
Maybe we should add a line like: "Each subdirecory tree of https://github.com/ceph/ceph-qa-suite constitutes a `suite`"